scripts/gen_feedback_data.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3.5
+
+#
+# Generate new feedback data for the NLC service given some initial training
+# data. This is to have more data available for the model that will reinforce
+# certain relationships. Since we have very little data this may help us (or may
+# not). For each training example in the seed data, a duplicate is made with
+# probability p with no capitalization and no punctuation. After this, n random
```

Comment:
Why do we need a probability here?

Reply:
I didn't want to duplicate everything, but just some items. I'm thinking of something like bootstrapping, where we generate new data from an original seed dataset.

Reply:
What do you think about regularizing user data before the call and then making the training data all have no capitalization or punctuation? I don't like the idea of having this probability distribution over the already black box behavior from Watson. But it doesn't seem like we'll be able to really test if it's doing well using either approach unless we could generate a large amount of interaction with our website.

Reply:
We won't be testing to see which works better. I think doing both might be ideal, but removing regularization from this script.

Reply:
Closing as per discussion.